x/gov/types/v1beta2/proposal.go

```diff
@@ -30,12 +30,13 @@ func NewProposal(messages []sdk.Msg, id uint64, submitTime, depositEndTime time.
 		return Proposal{}, err
 	}
 
+	tally := EmptyTallyResult()
+
 	p := Proposal{
 		ProposalId:       id,
 		Messages:         msgs,
 		Status:           StatusDepositPeriod,
-		FinalTallyResult: EmptyTallyResult(),
-		TotalDeposit:     sdk.NewCoins(),
```

Comment:
why removing `TotalDeposit`?

Reply:
It's less optimal to me to allocate an empty slice for TotalDeposit when we can just use nil